lib/constants.js

```diff
@@ -0,0 +1,112 @@
+/** @constant */
+export const DEFAULT_HEADERS = {
+  // Ensures that the page can't be displayed in a frame,
+  // regardless of where the request came from.
+  // It's a security feature to prevent "clickjacking" attacks,
+  // where an attacker might use an iframe to overlay a legitimate page over a deceptive page.
+  'X-Frame-Options': 'deny',
+  // The browser should enable the XSS filter and if a XSS attack is detected,
+  // rather than sanitizing the page, the browser will prevent rendering of the page entirely.
+  'X-XSS-Protection': '1; mode=block',
+  // Trust what the server says and not to perform MIME type sniffing.
+  // This can prevent certain security risks where the browser might interpret
+  // files as a different MIME type, which can be exploited in attacks.
+  'X-Content-Type-Options': 'nosniff',
+  // By default, do not load content from any source (default-src 'none').
+  // Images can be loaded from data: URLs (like Base64 encoded images).
+  // Styles can be loaded inline (style-src 'unsafe-inline'),
+  // which usually means from within the HTML itself rather than from external files.
```

Comment:
```suggestion
  /**
   * By default, do not load content from any source (`default-src 'none'`),
   * images can be loaded from `data:` URLs (like Base64 encoded images),
   * and styles can be loaded inline (`style-src 'unsafe-inline'`),
   * which usually means from within the HTML itself rather than from external
   * files.
   */
```

Docs, and use markdown code for code